testsuite/MDAnalysisTests/lib/test_distances.py

```diff
@@ -43,14 +43,6 @@ def test_transform_StoR_pass(coord_dtype):
     assert_equal(original_r, test_r)
 
 
-def test_transform_StoR_fail():
```

Comment:
Why removing this test?

Reply:
The test is included in `test_transform_StoR_pass` using parametrize. I guess someone modified it recently which got reflected here after I performed a pull from the develop branch.